src/openrct2/util/Util.h

```diff
@@ -18,42 +18,42 @@
 #include <type_traits>
 #include <vector>
 
-int32_t squaredmetres_to_squaredfeet(int32_t squaredMetres);
-int32_t metres_to_feet(int32_t metres);
-int32_t mph_to_kmph(int32_t mph);
-int32_t mph_to_dmps(int32_t mph);
+int32_t SquaredMetresToSquaredFeet(int32_t squaredMetres);
+int32_t MetresToFeet(int32_t metres);
+int32_t MphToKmph(int32_t mph);
+int32_t MphToDmps(int32_t mph);
 
-bool sse41_available();
-bool avx2_available();
+bool SSE41Available();
+bool AVX2Available();
 
-int32_t bitscanforward(int32_t source);
-int32_t bitscanforward(int64_t source);
-void bitcount_init();
-int32_t bitcount(uint32_t source);
-int32_t strlogicalcmp(char const* a, char const* b);
-char* safe_strcpy(char* destination, const char* source, size_t num);
-char* safe_strcat(char* destination, const char* source, size_t size);
+int32_t UtilBitScanForward(int32_t source);
+int32_t UtilBitScanForward(int64_t source);
+void BitCountInit();
+int32_t BitCount(uint32_t source);
+int32_t StrLogicalCmp(char const* a, char const* b);
+char* SafeStrCpy(char* destination, const char* source, size_t num);
+char* SafeStrCat(char* destination, const char* source, size_t size);
 
-uint32_t util_rand();
-float util_rand_normal_distributed();
+uint32_t UtilRand();
+float UtilRandNormalDistributed();
 
-bool util_gzip_compress(FILE* source, FILE* dest);
+bool UtilGzipCompress(FILE* source, FILE* dest);
 std::vector<uint8_t> Gzip(const void* data, const size_t dataLen);
 std::vector<uint8_t> Ungzip(const void* data, const size_t dataLen);
 
-int8_t add_clamp_int8_t(int8_t value, int8_t value_to_add);
-int16_t add_clamp_int16_t(int16_t value, int16_t value_to_add);
-int32_t add_clamp_int32_t(int32_t value, int32_t value_to_add);
-int64_t add_clamp_int64_t(int64_t value, int64_t value_to_add);
-money16 add_clamp_money16(money16 value, money16 value_to_add);
-money32 add_clamp_money32(money32 value, money32 value_to_add);
-money64 add_clamp_money64(money64 value, money64 value_to_add);
+int8_t AddClamp_int8_t(int8_t value, int8_t value_to_add);
```

Comment:
I thought this was a bit odd but its probably the best that can be done without making it a template.